core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -287,15 +287,15 @@ private static boolean isCreate(UpdateTableRequest request) {
     boolean isCreate =
         request.requirements().stream()
             .anyMatch(
-                UpdateTableRequest.UpdateRequirement.AssertTableDoesNotExist.class::isInstance);
+                org.apache.iceberg.UpdateRequirement.AssertTableDoesNotExist.class::isInstance);
```

Comment:
Can this not be imported?

Reply:
I manually added this, thinking it would conflict with the old `UpdateRequirement` but imported this now properly.